src/runtime/auto-prompt-manager-test.js

```diff
@@ -132,6 +132,9 @@ describes.realWin('AutoPromptManager', (env) => {
     );
     entitlementsManagerMock = sandbox.mock(entitlementsManager);
     sandbox.stub(deps, 'entitlementsManager').returns(entitlementsManager);
+    sandbox
```

Comment:
There's a redundant implementation of the exp flag logic @ https://github.com/subscriptions-project/swg-js/blob/667a03d00f4927d68c0aeec075e087ad4a08555f/src/runtime/auto-prompt-manager.ts#L1153, it should be removed in favor of the method in entitlements-manager.

Reply:
Removed.